Marlin/src/pins/pinsDebug_list.h

```diff
@@ -74,6 +74,16 @@
     REPORT_NAME_ANALOG(__LINE__, MAIN_VOLTAGE_MEASURE_PIN)
   #endif
 #endif
+#if PIN_EXISTS(POWER_MONITOR_CURRENT)
+  #if ANALOG_OK(POWER_MONITOR_CURRENT_PIN)
```

Comment:
Note to future visitors to this page. The C++ preprocessor always evaluates everything in an `#if` line, so `ANALOG_OK` is nested here to suppress ugly warnings when the pin isn't defined.